src/Draco.Compiler/Internal/Symbols/Symbol.cs

```diff
@@ -142,6 +142,14 @@ public virtual string FullName
     /// </summary>
     public virtual ImmutableArray<TypeSymbol> GenericArguments => ImmutableArray<TypeSymbol>.Empty;
 
+    /// <summary>
+    /// Compares signatures of this symbol and <paramref name="other"/> symbol.
+    /// </summary>
+    /// <param name="other">The other symbol to compare signatures with.</param>
+    /// <returns>True, if signature of this symbol and <paramref name="other"/> symbol are the same, otherwise false.</returns>
+    /// <exception cref="System.NotSupportedException"></exception>
+    public virtual bool SignatureEquals(Symbol other) => this.Name == other.Name;
```

Comment:
Shouldn't only overridable symbols have this? Do we have other uses?

Reply:
Check, if we still need this, since I have a hunch we could replace with CanBeOverridenBy where we use this. Not sure tho